week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -12,55 +12,75 @@ There are a few steps to being able view the future though:
 The question can be anything, but the answers are fixed,
 and have different levels of positivity or negativity.
 
-Below are the possible answers:
-
-## Very positive
-It is certain.
-It is decidedly so.
-Without a doubt.
-Yes - definitely.
-You may rely on it.
-
-## Positive
-As I see it, yes.
-Most likely.
-Outlook good.
-Yes.
-Signs point to yes.
-
-## Negative
-Reply hazy, try again.
-Ask again later.
-Better not tell you now.
-Cannot predict now.
-Concentrate and ask again.
-
-## Very negative
-Don't count on it.
-My reply is no.
-My sources say no.
-Outlook not so good.
-Very doubtful.
-*/
+Below are the possible answers:*/
+
+let answer = [
+  // ## Very positive
+  "It is certain.",
+  "It is decidedly so.",
+  "Without a doubt.",
+  "Yes - definitely.",
+  "You may rely on it.",
+  // ## Positive
+  "As I see it, yes.",
+  "Most likely.",
+  "Outlook good.",
+  "Yes.",
+  "Signs point to yes.",
+
+  // ## Negative
+  "Reply hazy, try again.",
+  "Ask again later.",
+  "Better not tell you now.",
+  "Cannot predict now.",
+  "Concentrate and ask again.",
+
+  // ## Very negative
+  "Don't count on it.",
+  "My reply is no.",
+  "My sources say no.",
+  "Outlook not so good.",
+  "Very doubtful.",
+];
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+function shakeBall() {
+  let number = Math.floor(Math.random() * answer.length);
+  console.log("The ball has shaken!");
+
+  return answer[number];
+}
 
 // The answer should come from shaking the ball
-let answer;
+// let answer;
 
 // When checking the answer, we should tell someone if the answer is
 // - very positive
 // - positive
 // - negative
 // - very negative
-function checkAnswer() {}
+function checkAnswer() {
+  if ((answer == 0, (answer = 1), (answer = 2), (answer = 3))) {
```

Comment:
The syntax isn't correct here. The correct syntax is:
if (condition) {
  lines of code to be executed if condition is true
}
A possible solution could start as:
if answer <= 4) {
  return (answer = "very positive");
}